CONTRIBUTING_A_NEW_QUEST.md

```diff
@@ -0,0 +1,390 @@
+For code style and more general info - see [CONTRIBUTING file](CONTRIBUTING.md#development.md).
+
+If you want to contribute code to the StreetComplete then making a new quest is one of the easiest programming tasks. Implementing a quest with interface design matching an existing one is quite simple.
+
+Contributions like that are highly welcomed and you would make mapping one more thing in OSM much easier!
+
+Reading the text below is not necessary to create a new quest - a basic method of duplicating existing ones and modifying them may be sufficient. And people were creating new quests before this documentation existed.
+
+But these materials may help or be quicker than trying to explore how things work fully on your own.
+
+# What is not required
+
+- expert programming ability - you do not need to be an expert programmer to make contributions, basic programming abilities are sufficient to implement a quest similar to existing ones
+- any Kotlin-specific or Android-specific knowledge - it is fine if you never did anything with Kotlin or Android
+- any Github specific knowledge
+
+Please [open an issue](https://github.com/streetcomplete/StreetComplete/issues/new) if you are stuck despite this. 
+
+# Required
+
+- programming ability - it is not a good first task for someone who never programmed anything. 
+  - Setup of environment takes long
+  - Editing even the simplest quest requires edits to at least two different files
+  - Kotlin familiarity is not needed, but the ability to adjust to a new syntax is needed
+  - It is expected that someone trying this can search for a solution to a typical problems on encountering an error message
+- familiarity with StreetComplete as a user - it is really recommended to be familiar with how StreetComplete works. Making hundreds of edits is not mandatory. But passing familiarity with how surveying with StreetComplete works is needed.
+- time - Android Studio setup is sadly complicated
+  - Multiple hours are typical, though most spend on waiting for various downloads
+  - setting up Android Studio will likely be more complex than writing the code
+- basic ability to use git (but as mentioned, anything GitHub specific will be explained) 
+- GitHub account
+- around 15GB of a free space
+  - the initial setup will download half of the Internet while getting emulators and dependencies. This is typical.
+
+# Dependencies - initial setup
+
+- install necessary software (Android Studio and git)
+- clone StreetComplete repository
+- open StreetComplete in Android Studio
+- [setup an emulator in Android Studio](https://developer.android.com/studio/run/emulator#install) (you can also connect to a real device via usb, this will not be later mentioned but can fully replace the emulator)
```

Comment:
```suggestion
- [setup an emulator in Android Studio](https://developer.android.com/studio/run/emulator#install) (You can also connect to a real device via USB. This can fully replace the emulator, but this document won't go into more detail.)
```

Reply:
Here linking some docs (official ones preferably) would be nice